src/core/Akka.Remote.Tests/RemotingSpec.cs

```diff
@@ -20,10 +21,10 @@
 
 namespace Akka.Remote.Tests
 {
-    
+
     public class RemotingSpec : AkkaSpec
     {
-        public RemotingSpec():base(GetConfig())
+        public RemotingSpec() : base(GetConfig())
```

Comment:
Can't review it properly. There is lack of specs. I think if we haven't ported or won't ported any spec (or part of spec) to Akka.NET we should write a comment about it. Right not I don't know - this spec is outdated or not